context/app/static/js/components/Search/Results/Results.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ViewSwitcherHits } from 'searchkit'; // eslint-disable-line import/no-duplicates
+
+import ResultsTable from '../ResultsTable';
+import ResultsTiles from '../ResultsTiles';
+
+function Results(props) {
+  const { sortOptions, hitsPerPage, resultFields, detailsUrlPrefix, idField, resultFieldIds, type } = props;
+
+  return (
```

Comment:
```suggestion
  // Both the table and tiles are rendered, no matter the view.
  // Conditionally rendering one or the other is complicated because ...
  return (
```
If you can remind our future selves why it's being done this way. Thanks!

Reply:
Only one of the sort components must stay mounted (the sort dropdown). I'll add a comment to reflect that.